test/unittest/rtps/builtin/BuiltinDataSerializationTests.cpp

```diff
@@ -12,6 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include "fastdds/dds/log/Log.hpp"
```

Comment:
This header is not used directly in this source file, so I suggest removing it. If there is any build issue, then the proper file which is missing this include is the one to look for.